compiler/src/codegen/compcore.re

```diff
@@ -3434,6 +3516,12 @@ let compile_wasm_module =
     } else {
       default_features;
     };
+  let features =
+    if (Config.simd^) {
+      [Module.Feature.simd128, ...features];
```

Comment:
Binaryen's default behaviour is to error if a feature isn't enabled but the code is used right?